### ISSUE TITLE
[overlay|overlay-trigger] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1284]
### Feedback

❌ ARC - non active element in tab order (sp-active-overlay)
- The example does not move keyboard focus correctly. I would assume that this should happen automatically where focus moved from the trigger to the content when opened - and back when closed, but this does not seem to occur in the example (review internally, should be fixed)
❌ active-overlay has tabindex=0 but no role or accessible name
❌ there is no way for a keyboard user to close the overlay in this example (understood that this is an example - but should meet a11y guidelines)
❌ The button opens the popover but focus does not move, so nothing is read.

### Path forward

1. Let's review how to get that result in the ARC tooling so I can understand what this means to the UI.
2. This is due to not having a specific plan for this content and having a longer standing "default" that is not accessible. Make a 'breaking' change to set the default to something that "works" even if it's only in specific use cases.
3. This is related to the above, will need revisit after that work is done.
4. Is `escape` enough of an option here, or do we need to add more? This has been added since your view. Or do we need to add a "hidden" button?
5. Related to no. 2 above.

### Notice:

This content has since been expanded to:
https://opensource.adobe.com/spectrum-web-components/components/overlay
https://opensource.adobe.com/spectrum-web-components/components/overlay-trigger

cc @jnurthen

**Comment by Westbrook** [comment]
Not sure how much of this is still current, but good information to keep in mind as we move to revisit this API surface.

**Comment by Westbrook** [comment]
This will no longer be based on life code as of the next couple of weeks, so I'm closing in favor of an updated review.
